deposit/src/main/java/edu/unc/lib/deposit/work/AbstractDepositJob.java

```diff
@@ -232,68 +208,24 @@ protected void verifyRunning() {
 	}
 
 	/**
-	 * Returns the PREMIS events file for the given PID
-	 *
-	 * @param pid
-	 * @return
-	 */
-	protected File getEventsFile(PID pid) {
-		return createOrAppendToEventsFile(pid, null);
-	}
-
-	/**
-	 * Appends an event to the PREMIS log for the given pid. If the log does not exist, it is created
-	 *
-	 * @param pid
-	 * @param event
-	 */
-	protected void appendDepositEvent(PID pid, Element event) {
-		createOrAppendToEventsFile(pid, event);
-	}
-
-	/**
 	 * Appends an event to the PREMIS document for the given PID, creating the document if it does not already exist or
 	 * is corrupt.
 	 *
 	 * @param pid
-	 * @param event
 	 * @return the premis document file
 	 */
-	private File createOrAppendToEventsFile(PID pid, Element event) {
-		File file = new File(depositDirectory, DepositConstants.EVENTS_DIR + "/" + pid.getUUID() + ".xml");
+	public PremisLogger createOrAppendToEventsFile(PID pid) {
```

Comment:
This method doesn't do any appending anymore, how about renaming it to "getPremisLogger"?